third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -2586,6 +2586,72 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("correction"),
       py::arg("keepdim") = false,
       py::return_value_policy::reference);
+  nvf_ops.def(
```

Comment:
Just want to point out here that, uniform & normal are not tensor methods, so we don't have to repeat it for that.
But the duplication is less than ideal cc'ing @jacobhinkle  :wink:

Reply:
Unless we want `randn_like`...

Reply:
Yeah, we can revisit that at a later point~ for now I'm just going to enjoy the convenience of the new API :)